builder/alicloud/ecs/step_create_alicloud_instance.go

```diff
@@ -0,0 +1,198 @@
+package ecs
+
+import (
+	"fmt"
+	"github.com/denverdino/aliyungo/common"
+	"github.com/denverdino/aliyungo/ecs"
+	"github.com/mitchellh/multistep"
+	"github.com/mitchellh/packer/packer"
+	"io/ioutil"
+	"log"
+	"strings"
+)
+
+const (
+	shebang      = "#!/bin/sh\n"
+	key_function = `
+export ssh_directory="/root/.ssh"
```

Comment:
Currently, this is a limitation and this assumption is correct and it will be removed after the key pair function work.